proposed/http-client/http-client-meta.md

```diff
@@ -33,6 +33,11 @@ client will allow libraries to be decoupled from an implementation such as Guzzl
 specify the default behaviours.
 * The purpose is not to be opinionated about the use of middlewares (PSR-15).
 
+#### Asynchronous HTTP client
+
+The reason asynchronous support is a non-goal is because a PSR for HTTP clients 
+should not define Promises. That should be a separate PSR. Waiting for a such PSR 
```

Comment:
Of course it doesn't say it has to be blocking, as that's the default for any IO operation in PHP. It has to be blocking, because the interface requires an immediate return on a call to these methods.

> Im saying that you can implement PSR-7 with non-blocking.

Could you try to elaborate how that should be possible? [See what React did.](https://github.com/reactphp/http/blob/ad45e03c17fbf3ec6d8a5bf77d3000a8f5f40f83/src/HttpBodyStream.php#L80-L150)